book.rb

```diff
@@ -0,0 +1,15 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental=(rental)
+    @rentals.push(rental)
+    rental.book = self
+  end
```

Comment:
- [ ]  For `add_rental`for book you should call the parameters `person, date`, and then the `book itself should be sent as a parameter to create the new rental achieving the 'has-many' association 🙃.
It can be similar to:

```ruby
  def add_rental(person, date)
    Rental.new(date, self, person)
  end
```